src/Atc.Network/Enums/IPProtocolType.cs

```diff
@@ -0,0 +1,14 @@
+// ReSharper disable once CheckNamespace
+namespace Atc.Network;
+
+public enum IPProtocolType
+{
+    None,
```

Comment:
Wouldn't it make sense to assign the default port numbers to these values?
Example: 

```
Http = 80,
Https = 443
```

Reply:
How is TCP in the same category as HTTP ?